modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/ContributorsBuilder.es.js

```diff
@@ -45,11 +42,9 @@ class ContributorBuilder extends React.Component {
 		onConjunctionChange: PropTypes.func,
 		onPreviewMembers: PropTypes.func,
 		onQueryChange: PropTypes.func,
-		portletNamespace: PropTypes.string.isRequired,
 		previewMembersURL: PropTypes.string,
 		propertyGroups: PropTypes.arrayOf(propertyGroupShape),
 		renderEmptyValuesErrors: PropTypes.bool,
-		scopeName: PropTypes.string,
```

Comment:
I believe that you can also remove the siteItemSelectorURL.

Reply:
ahh you're right, thank you!